CMakeLists.txt

```diff
@@ -123,6 +123,7 @@ option (USE_STATIC_BOOST                    "Use static Boost libraries"
 option (USE_STATIC_OIIO                     "Use static OpenImageIO libraries"                      ON)
 option (USE_STATIC_OCIO                     "Use static OpenColorIO libraries"                      ON)
 option (USE_STATIC_OSL                      "Use static OpenShadingLanguage libraries"              ON)
+option (USE_STATIC_EXR                      "Use static OpenEXR libraries"                          ON)
```

Comment:
Can you move this up to follow the same ordering as the `USE_EXTERNAL_*` options?